test/publishing_api_v2_test.rb

```diff
@@ -334,4 +334,230 @@ def content_item_for_content_id(content_id, attrs = {})
       assert_equal "Cannot publish an already published content item", error.error_details["error"]["message"]
     end
   end
+
+  describe "#get_links" do
+    describe "when there's a links entry with links" do
```

Comment:
:+1:, I like this split of pact setup into before and verification in the actual spec. For the sake of consistency, we should probably move the other tests in this file to this pattern.